CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
 - [2106](https://github.com/FuelLabs/fuel-core/pull/2106): Remove deadline clock in POA and replace with tokio time functions.
+- [2112](https://github.com/FuelLabs/fuel-core/pull/2112): Alter the way the sealed blocks are fetched with a given height.
```

Comment:
We need to move it to `## [Unreleased]` section

Reply:
addressed in https://github.com/FuelLabs/fuel-core/pull/2112/commits/da650f4ebe72bb42003d859750eb84cbf26a2271